src/webapp/src/characters/fetchCharacters.tsx

```diff
@@ -0,0 +1,21 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+import { ApiKnownError, Character } from './types';
+
+export const fetchCharacters = createAsyncThunk<
+  Character[],
+  void,
+  { rejectValue: ApiKnownError }
+>('characters/getCharacters', async (_, { rejectWithValue }) => {
```

Comment:
```suggestion
>('characters/fetchCharacters', async (_, { rejectWithValue }) => {
```

Reply:
Of course! I have to think about those things whenever I make the changes. I need a better process to review everything I touch and not make this silly mistakes. Thanks